src/bench/bench.cpp

```diff
@@ -41,18 +42,44 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 } // namespace
 
-benchmark::BenchRunner::BenchmarkMap& benchmark::BenchRunner::benchmarks()
+namespace benchmark {
+
+std::map<std::string, PriorityLevel> map_priority_level = {
+    std::make_pair("high", PriorityLevel::HIGH),
+    std::make_pair("medium", PriorityLevel::MEDIUM),
+    std::make_pair("low", PriorityLevel::LOW)
+};
+
+std::string ListPriorities()
+{
+    std::string tags;
+    for (const auto& [name, level] : map_priority_level) {
+        tags += name + ",";
+    }
+    tags.pop_back(); // remove the last comma
+    return tags;
+}
```

Comment:
With `#include <util/string.h>`:
```suggestion
{
    return Join(map_priority, ',', [](const auto& entry){  return entry.first; });
}
```

Reply:
Not a huge fan of how this sorts alphabetically by label instead of by increasing priority, but... not a very straightforward around that it seems. Just raising in case you have ideas.

Reply:
> Not a huge fan of how this sorts alphabetically by label instead of by increasing priority, but... not a very straightforward way around that it seems. Just raising in case you have ideas.

Could add a custom comparator. One sec.